content/events/2022-02-2nd-outreachy-presentation/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: " Outreachy Intern Presentation : Community Onboarding Project"
+date: '2022-03-03'
+days: 1
+tease: ''
+continent: GL
+location: 'Online, Global'
+image: "https://www.outreachy.org/static/css/img/logo-no-background.86e64a85437c.svg"
+location_url: "/events/outreachy-presentation/"
+external_url:
```

Comment:
Do you erally need to set location_url? This looks ood to me.

Reply:
I'm not sure, I've copied the other event template but now revising other templates it isn't used. I'll delete it